unsatisfied peer-dependency on typescript in v13.2.0 of `@commitlint/cli`

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior

<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->

running `npm ls` with `@commitlint/cli` installed should exit zero, meaning that there are no unsatisfied peer dependencies in the dependency tree.

## Current Behavior

<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

`npm ls` reports an unsatisfied peer dependency on `typescript`:

```
npm ERR! peer dep missing: typescript@>=2.7, required by ts-node@9.1.1
```

`npm ls ts-node` shows that the requirement is pulled in by `@endemolshinegroup/cosmiconfig-typescript-loader`:

```
  └─┬ @commitlint/cli@13.2.0
    └─┬ @commitlint/load@13.2.0
      └─┬ @endemolshinegroup/cosmiconfig-typescript-loader@3.0.2
        └── ts-node@9.1.1 
```

## Affected packages

- [x] cli
- [ ] core
- [ ] prompt
- [ ] config-angular

## Possible Solution

<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

either make `@endemolshinegroup/cosmiconfig-typescript-loader` an optional peer dependency or define typescript as a direct dependency of the cli package in order to satisfy the peer requirement.

## Steps to Reproduce (for bugs)

<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->

1. `npm install @commitlint/cli -D`
2. `npm ls`


## Context

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

i use commitlint in many projects that are not typescript projects, so i do not want to define typescript as a direct dependency in those projects to satisfy a peer requirement of commitlint.

## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

|             Executable | Version |
| ---------------------: | :------ |
| `commitlint --version` | 13.2.0  |
|        `git --version` | 2.27.0  |
|       `node --version` | 14.17.5 |
|        `npm --version` | 6.14.14 |

--- a/@commitlint/load/package.json
+++ b/@commitlint/load/package.json
@@ -48,7 +48,8 @@
     "chalk": "^4.0.0",
     "cosmiconfig": "^7.0.0",
     "lodash": "^4.17.19",
-    "resolve-from": "^5.0.0"
+    "resolve-from": "^5.0.0",
+    "typescript": "^4.4.3"
   },
   "gitHead": "70f7f4688b51774e7ac5e40e896cdaa3f132b2bc"
 }

